specifications/dbus.md

```diff
@@ -1,5 +1,7 @@
 # Specifications
 
+UnifiedPush Spec: DBUS_2.0.0
```

Comment:
I had assumed the dbus spec was on 0.x until some adoption and feedback from Linux app devs. I'm not really sure if it should be considered as 'final' as 1.0 or 2.0 signals right now. Maybe it can be 0.2.0?